gpkit/nomials/nomial_math.py

```diff
@@ -633,14 +633,27 @@ def as_posyslt1(self):
         "Returns the posys <= 1 representation of this constraint."
```

Comment:
I'm surprised so much substitution is happening here. @bqpd let's discuss in person